spec/features/admin/order_cycles_spec.rb

```diff
@@ -58,6 +58,38 @@
     end
   end
 
+  describe 'listing order cycles with other locales' do
+    let!(:oc_de) { create(:simple_order_cycle, name: 'oc', orders_open_at: '2012-01-01 00:00:00') }
+
+    before { I18n.locale = :de }
+    after { I18n.locale = :en }
```

Comment:
IMO for these cases an [around hook](https://relishapp.com/rspec/rspec-core/v/2-0/docs/hooks/around-hooks)  is better suited